modules/auxiliary/admin/dcerpc/samr_computer.rb

```diff
@@ -17,7 +17,7 @@ def initialize(info = {})
         info,
         'Name' => 'SAMR Computer Management',
         'Description' => %q{
-          Add, lookup and delete computer accounts via MS-SAMR. By default
+          Add, lookup and delete computer / machine accounts via MS-SAMR. By default
```

Comment:
Just some search engine optimisation here, as I've never been able to find this module with the `search` command on the first search